### PR TITLE
fix: exit if odigos is already installed

### PR DESCRIPTION
**Pull request by HeetVekariya**
# Fixes issue:
- Issue: #660 
- Update the error message for install of odigos if it is already installed

<br>

# Changes made:
- Updated the code such that if  ```Namepace``` is found then print the updated error message and exit

<br>

# Files changed:
- **install.go** (cli/cmd/install.go)

### CODE REVIEW

**Comment by HeetVekariya** [comment]
- @blumamir Test cases:

1) If already installed:
Screenshot from 2023-10-20 10-49-49

2)  if not installed:
Screenshot from 2023-10-20 10-50-40

- I found that the code goes in the infinite loop due to this code [here](https://github.com/keyval-dev/odigos/blob/3319cab6c2ea6034d3aa25ca073a1dd8602339a3/cli/cmd/install.go#L89):
```go
if !skipWait {
  l := log.Print("Waiting for Odigos pods to be ready ...")
  err := wait.PollImmediate(1*time.Second, 3*time.Minute, arePodsReady(ctx, client, ns))
  if err != nil {
     l.Error(err)
  }
  
  l.Success()
}
```

- This calls the function ```arePodsReady(ctx, client, ns```, whose function definition is:

```go
func arePodsReady(ctx context.Context, client *kube.Client, ns string) func() (bool, error) {
  return func() (bool, error) {
  pods, err := client.CoreV1().Pods(ns).List(ctx, metav1.ListOptions{})
  if err != nil {
	  return false, err
  }
  runningPods := 0
  for _, p := range pods.Items {
	  if p.Status.Phase == corev1.PodFailed {
		  return false, fmt.Errorf("pod %s failed", p.Name)
	  }
  
	  if p.Status.Phase == corev1.PodRunning {
		  runningPods++
	  }
  }
  
  return runningPods == len(pods.Items), nil
  }
}
```
- Here the for loop does not terminates.
- Is this problem already exists or my code has created this problem ?

File: cli/cmd/install.go [hunk 1]
```diff
@@ -54,6 +55,12 @@ This command will install k8s components that will auto-instrument your applicat
 		cmd.Flags().StringSliceVar(&ignoredNamespaces, "ignore-namespace", DefaultIgnoredNamespaces, "--ignore-namespace foo logging")
 		fmt.Printf("Installing Odigos version %s in namespace %s ...\n", versionFlag, ns)
 
+		nsflag, err := resources.GetOdigosNamespace(client, ctx)
```
Comment (blumamir) [inline cli/cmd/install.go:58]: nit: this ns has nothing to do with any flag.
```suggestion
		existingOdigosNs, err := resources.GetOdigosNamespace(client, ctx)
```

**Review by blumamir — APPROVED** [review]

File: cli/cmd/install.go (hunk 1 above)
Comment (HeetVekariya) [reply cli/cmd/install.go]: Changed name to `existingOdigosNs`

**Comment by HeetVekariya** [comment]
@blumamir I have made the suggested changes

**Comment by HeetVekariya** [comment]
Thank you for merging

**Pull request merged**